### PR TITLE
Dota 2: Fixes #3296 Skeleton King Trigger not working

### LINKED ISSUE #3296: Dota2 IA - Skeleton King Trigger not working
### Description

Dota2 IA - When ostarion is searched following error occurs. Same errors occur when it is searched as

- Dota 2 SkeletonKing
-Dota 2 Skeleton King
-Dota2 Skeleton King
-Dota2 Ostarion
-Dota2 SkeletonKing
**EDIT:** Dota2 Wrath king is not working in addition.

ekran goruntusu 79

## Steps to recreate

Search for any of the queries mentioned above.

## People to notify

@moollaza 
@echosa 

## Resources
- Join [DuckDuckHack Slack](https://quackslack.herokuapp.com/) to ask questions
- Join the [DuckDuckHack Forum](https://forum.duckduckhack.com/) to discuss project planning and Instant Answer metrics
- Read the [DuckDuckHack Documentation](https://docs.duckduckhack.com/) for technical help

---

Instant Answer Page: https://duck.co/ia/view/dota2

### PR DESCRIPTION
**Pull request by ManrajGrover**
## Description of new Instant Answer, or changes
The API response must have changed and now spaces between bigrams are being padded with underscores. This fixes the search result which was currently being used directly.

## Related Issues and Discussions
Fixes #3296

## People to notify
@pjhampton @echosa

---

Instant Answer Page: https://duck.co/ia/view/dota2

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by ManrajGrover** [comment]
@pjhampton This can be deployed to beta since I feel it has been fixed 😄

**Comment by pjhampton** [comment]
Thanks @ManrajGrover. Only @moollaza can deploy spices/goodies to beta. I'm sure he will get back on it soon 😄

File: share/spice/dota2/dota2.js [hunk 1]
```diff
@@ -179,10 +181,12 @@
         }
 
         var searchTarget = DDG.get_query().toLowerCase().replace(/dota\s?2/, "").trim();
+        var keyTarget = searchTarget.split(' ').join('_');
```
Comment (moollaza) [inline share/spice/dota2/dota2.js]: why not use replace?

**Comment by moollaza** [comment]
@ManrajGrover Thanks! This LGTM :+1:

**Pull request merged**